elliottlib/errata.py

```diff
@@ -642,6 +642,7 @@ def add_jira_bugs_with_retry(advisory_id: int, bugids: List[str], noop: bool = F
             attached_bugs = re.findall("Issue (.*) The issue is filed already in", str(e))
             if attached_bugs:
                 chunk_of_bugs = [b for b in chunk_of_bugs if b not in [b.upper() for b in attached_bugs]]
+                advisory = Erratum(errata_id=advisory_id)
```

Comment:
We can instead do
```
advisory.removeJiraIssues(attached_bugs)
advisory.commit()
```

Reply:
let's reset the object and fix it errata_tool